L1Trigger/TrackFindingTracklet/interface/CleanTrackMemory.h

```diff
@@ -0,0 +1,141 @@
+//This class stores the track fit
+#ifndef CLEANTRACKMEMORY_H
+#define CLEANTRACKMEMORY_H
+
+#include "Tracklet.h"
+#include "MemoryBase.h"
+
+using namespace std;
```

Comment:
Please don't `using namespace std` in a header file.

Reply:
fixed by migrating to h/cc files